### PR TITLE
Setup testing farm

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
- Add .fmf and setup testing-farm.

---

TODO:
- [x] add some real tests

### CODE REVIEW

**Comment by TomasTomecek** [comment]
hm, what's wronk with rawhide?
```
# pwd; cd ~/git-source && pwd && make check[11:42:48] [E] [cruncher] [/ci/test/build/full] Command 'cd ~/git-source; pwd; cd ~/git-source && pwd && make check' failed with return code '255'
```
doesn't tell me anything really

/summon @thrix :D

File: .packit.yaml [hunk 1]
```diff
@@ -29,5 +29,14 @@ jobs:
...
     targets:
     - fedora-29-x86_64
+    - fedora-30-x86_64
+    - fedora-31-x86_64
     - fedora-rawhide-x86_64
+- job: tests
+  trigger: pull_request
+  metadata:
+    targets:
+    - fedora-29-x86_64
```
Comment (jpopelka) [inline .packit.yaml]: https://www.linuxcompatible.org/story/fedora-29-eol-on-20191126/

Comment (lachmanfrantisek) [reply .packit.yaml]: So we have another 6 days..;) I'll remove it.

**Comment by thrix** [comment]
looking

**Pull request merged**